app_radio_module/src/transmitter.c

```diff
@@ -42,6 +45,12 @@ l_udp_socket_list_t udp_socket_list = {
 // Queues
 K_MSGQ_DEFINE(lora_tx_queue, sizeof(l_lora_packet_t), CONFIG_LORA_TX_QUEUE_SIZE, 1);
 
+// Timers
+struct k_timer gnss_tx_timer;
+static void gnss_tx_on_expire(struct k_timer *timer_id); // Forward Declaration
+K_TIMER_DEFINE(gnss_tx_timer, gnss_tx_on_expire, NULL);
+static bool ready_to_tx = false;
```

Comment:
NIT: Move static bool to the top with the other static vars

Reply:
resolved in https://github.com/RIT-Launch-Initiative/FSW/pull/85/commits/9bb8c95ad3ac067dbd364b03f1374f5254d74fad